beacon-chain/sync/initial-sync/blocks_fetcher_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	p2pm "github.com/prysmaticlabs/prysm/beacon-chain/p2p"
 	p2pt "github.com/prysmaticlabs/prysm/beacon-chain/p2p/testing"
 	beaconsync "github.com/prysmaticlabs/prysm/beacon-chain/sync"
+	prysmsync "github.com/prysmaticlabs/prysm/beacon-chain/sync"
```

Comment:
there is also `beaconsync`? why 2 imports?

Reply:
Copy-paste issue (as this refactoring is moving methods, got their deps moved as is as well). My bad, wonder why none of the static analyzers located it. Will fix now.

Reply:
Should be fixed now.